### PR TITLE
python-package: fix version

### PR DESCRIPTION
**Pull request by henry0312**

### CODE REVIEW

File: python-package/setup.py [hunk 1]
```diff
@@ -169,6 +172,6 @@ def run(self):
           },
           packages=find_packages(),
           include_package_data=True,
-          data_files=data_files,
```
Comment (guolinke) [inline python-package/setup.py]: why change this?

Comment (henry0312) [inline python-package/setup.py]: this is not used.

**Comment by henry0312** [comment]
Before:

```
❯ ipython
Python 3.6.1 (default, May 12 2017, 16:45:09)
Type 'copyright', 'credits' or 'license' for more information
IPython 6.0.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import lightgbm

In [2]: lightgbm.version
Out[2]: '2.0.1'
```

After:

```
❯ ipython
Python 3.6.1 (default, May 12 2017, 16:41:49)
Type 'copyright', 'credits' or 'license' for more information
IPython 6.0.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import lightgbm

In [2]: lightgbm.version
Out[2]: '2.0.5'
```

**Comment by guolinke** [comment]
@wxchan comments ?

File: python-package/lightgbm/__init__.py [hunk 2]
```diff
@@ -20,11 +20,12 @@
...
 
-version = '2.0.1'
-if os.path.isfile('./VERSION.txt'):
-    version = open('./VERSION.txt').read().strip()
 
-__version__ = version
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+if os.path.isfile(os.path.join(dir_path, 'VERSION.txt')):
+    version = open(os.path.join(dir_path, 'VERSION.txt')).read().strip()
```
Comment (wxchan) [inline python-package/lightgbm/__init__.py]: do we still need variable `version` now?

Comment (henry0312) [inline python-package/lightgbm/__init__.py]: I don't think it is not necessary, so I'll remove it.

**Comment by henry0312** [comment]
updated

**Pull request merged**